### PR TITLE
Add Spanish localization

### PR DESCRIPTION
**Pull request by hwaltman19**
This commit translates all JSON messages into universal Spanish equivalents.

### CODE REVIEW

File: _locales/es/messages.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "extensionName": {
+        "message": "Haz más preciso el texto \"No soy roboto\" de ReCaptcha",
```
Comment (strugee) [inline _locales/es/messages.json]: This looks like the same text as `extensionDescription`. They _are_ different, though only by one word... do the originals just translate to the exact same text?

Comment (hwaltman19) [reply _locales/es/messages.json]: They are not identical in either language, so I have corrected the extension name in a separate commit. Thank you for pointing out the difference!

Comment (strugee) [reply _locales/es/messages.json]: Awesome, thanks very much for the translation! LGTM :thumbsup:

**Review by strugee — APPROVED** [review]

**Comment by strugee** [comment]
@hwaltman19 I'm seeing some weird results with the extension's internationalization support locally and I'm having trouble debugging due to some dumb technical issues. Did you test these changes locally and make sure the ReCaptcha text was actually changed correctly? If not, would you mind doing so? I can post a development build and walk you through the process if you need. Thanks!

**Comment by hwaltman19** [comment]
@strugee Thanks for your report. I've loaded the extension as a temporary debug build in the English version of Firefox. If I log the output of `label.innerText`, I see the English text even if I've changed my locale in `about:config`. Right now, the localization only works when I test in a fully-translated Spanish version of Firefox, which is not ideal. I'll take a look and see if there is a way to change this.

**Comment by hwaltman19** [comment]
Here is what the translation looks like on a live page.
es_locale_success

**Comment by strugee** [comment]
@hwaltman19 yeah, that matches my observations. I also made the `about:config` change and noticed that the extension name in `about:debugging` changed, but not the ReCaptcha text. I figured maybe it would only show up if I changed the language pack... but I couldn't actually figure out how to do that under Debian and its packaging. \*sigh\*

Anyway, thank you for verifying! I wonder if it's an upstream Firefox bug? I'll look into it if I get some time.

**Pull request merged**